blog/model/add_entry_model.py

```diff
@@ -0,0 +1,39 @@
+import os
+from flask import request, flash
+from blog.database.db import WorkWithDatabase
+from blog.configurations.config import (STATIC_AUDIO_PATH, STATIC_IMAGES_PATH,
+                                                   IMG_HEIGHT_REQUIR, IMG_WIDTH_REQUIR,
+                                                   WATERMARK_OPACITY,WATERMARK_FILE,
+                                                   SQL_QUERIES_PATH)
+from blog.servises.img_resizer_and_watermark_add import img_resizer_and_watermark_add
+from datetime import datetime
+from werkzeug.utils import secure_filename
+
+
+class AddEntryViewModel():
+
+    def __init__(self,db = WorkWithDatabase):
+        self.db = db()
+
+    def add_entry_to_database(self):
+        audiofile = request.files['audiofile']
+        imagefile = request.files['imagefile']
+        timestamp = datetime.now().strftime("%Y%m%d-%H%M%S")
+        audiofilename = 'empty'
+        new_audiofilename = 'empty'
+        new_imagefilename = 'empty'
+        if audiofile:
+            audiofilename = secure_filename(audiofile.filename)
```

Comment:
Because I save it to database and decided that it would be safer than just put the original name there.